pages/common/pdfjoin.md

```diff
@@ -0,0 +1,15 @@
+# pdfjoin
+
+> PDF merging utility.
```

Comment:
Is this `pdfjoin` part of the [`pdfjam`] package by David Firth? Or another package?

[`pdfjam`]: http://go.warwick.ac.uk/pdfjam

Reply:
Looks like it is part of `pdfjam`:

```bash
$ pdfjoin --help | head

pdfjam is a shell-script front end to the LaTeX 'pdfpages' package (for
which, see http://www.ctan.org/tex-archive/macros/latex/contrib/pdfpages).

Usage: pdfjam [OPTIONS] [--] [FILE1 [SEL1]] [FILE2 [SEL2]]...
where.....
```